.github/workflows/check-source-signatures.yml

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+name: Source Signature Check
+
+on:
+  push:
+    branches: [3.0*]
```

Comment:
we can probably enable for all branches?

Reply:
It relies on the LKG, once we normalize the way to do repos I agree we should turn on.